src/react-components/hub-create-panel.js

```diff
@@ -214,6 +226,22 @@ class HubCreatePanel extends Component {
                             <span>{environmentAuthor.name}</span>
                           </span>
                         ))}
+                      {environmentAuthor &&
+                        environmentAuthor.organization &&
+                        (environmentAuthor.organization.url ? (
+                          <a
+                            href={environmentAuthor.organization.url}
+                            target="_blank"
```

Comment:
I'd remove the `target` (for [these reasons](https://css-tricks.com/use-target_blank/))